### PR TITLE
Added banner 

### LINKED ISSUE #38: 🚀 Feature Request: Add Grey Software sticker & Project logo
### Problem Overview 👁️‍🗨️ 

Users should be able to see the Grey Software sticker and the LinkedIn-Focus logo on the README.md file.

### What would you like? 🧰

We want the Grey Software sticker and the LinkedIn-Focus logo as the header on the README.md file. Below is an example. You would also need to add the Grey Software sticker next to it.

image

### What alternatives have you considered? 🔍

N/A

### Additional details ℹ️

The Linked-In Focus logo image can be found in this repository under the **src** folder. The file name is **icon.png**. Below is the image of the Grey Software sticker that should be used.

image

### PR DESCRIPTION
**Pull request by Owen000**
I added a banner with the grey software sticker and LinkedInFocus logo as requested. Fixes #38

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,12 @@
+<p align="center">
+    <img alt="LinkedIn-Focus icon" width="500" src="https://github.com/Owen000/LinkedIn-Focus/raw/master/src/LinkedInFocusBanner.png">
```
Comment (ArsalaBangash) [inline README.md:2]: This source is tied to your fork. Let's have it reference the repo's icon instead: https://github.com/grey-software/LinkedIn-Focus/blob/master/src/icon.png

File: README.md [hunk 2]
```diff
@@ -1,4 +1,12 @@
+<p align="center">
```
Comment (ArsalaBangash) [inline README.md:1]: Does this still work with a div? A div tag would technically be the correct one to use here.

**Review by ArsalaBangash — CHANGES_REQUESTED** [review]
Solid iteration!

Please address the review comments, and add the Grey software sticker at the end of the readme as per the issue descriptions' additional details. 

I went ahead and uploaded the sticker for you here: https://github.com/grey-software/LinkedIn-Focus/blob/master/grey-software-sticker.png

**Comment by Owen000** [comment]
Fixed all those issues. 👍

File: README.md [hunk 3]
```diff
@@ -12,4 +20,8 @@ Use [this guide](https://developer.chrome.com/extensions/getstarted) to load the
 ![Before](before.png)
 
 ### After 
-![After](after.png)
\ No newline at end of file
+![After](after.png)
+
+<div align="center">
+    <img alt="GreySoftware icon" width="100" src="https://github.com/grey-software/LinkedIn-Focus/blob/master/src/logo.png">
```
Comment (ArsalaBangash) [inline README.md:26]: Please use the sticker here https://github.com/grey-software/LinkedIn-Focus/blob/master/grey-software-sticker.png

**Review by ArsalaBangash — CHANGES_REQUESTED** [review]
Please add the sticker and remove the banner that you added :)

**Comment by Owen000** [comment]
Fixed

**Review by ArsalaBangash — APPROVED** [review]
Great work!

**Pull request merged**